packages/select/src/components/multi-select/multiSelect2.tsx

```diff
@@ -28,20 +28,29 @@ import {
     TagInputAddMethod,
     TagInputProps,
 } from "@blueprintjs/core";
-import { Popover2 } from "@blueprintjs/popover2";
+import { Popover2, PopupKind } from "@blueprintjs/popover2";
 
 import { Classes, IListItemsProps, SelectPopoverProps } from "../../common";
-import { IQueryListRendererProps, QueryList } from "../query-list/queryList";
+import { IQueryListRendererProps, QueryList, QueryListProps } from "../query-list/queryList";
 
 // N.B. selectedItems should really be a required prop, but is left optional for backwards compatibility
 
-export interface MultiSelect2Props<T> extends IListItemsProps<T>, SelectPopoverProps {
+export interface MultiSelect2Props<T>
+    extends IListItemsProps<T>,
+        Pick<QueryListProps<T>, "menuProps">,
+        SelectPopoverProps {
     /**
      * Whether the component should take up the full width of its container.
      * This overrides `popoverProps.fill` and `tagInputProps.fill`.
      */
     fill?: boolean;
 
+    /**
+     * Adds `id={listboxId}` to the list Menu and `aria-controls={listboxId}` to the
+     * combobox element that opens it
+     */
+    listboxId?: string;
```

Comment:
we should auto-generate this ID like `<Dialog>` does here, to simplify the API for consumers:

https://github.com/palantir/blueprint/blob/7ea317e054565a23dc4407f7feb5bef4553c195c/packages/core/src/components/dialog/dialog.tsx#L115-L116

then, we wouldn't need to add this `listboxId` prop.